pandas/tests/io/parser/compression.py

```diff
@@ -12,6 +12,10 @@
 import pandas.util.testing as tm
 import pandas.util._test_decorators as td
 
+import gzip
+import bz2
+lzma = compat.import_lzma()
```

Comment:
this is slightly annoying you have to skip that test if can't import lzma but you need the file reference
```
Traceback:
pandas/tests/io/parser/test_parsers.py:23: in <module>
    from .compression import CompressionTests
pandas/tests/io/parser/compression.py:17: in <module>
    lzma = compat.import_lzma()
pandas/compat/__init__.py:346: in import_lzma
    from backports import lzma
E   ImportError: cannot import name lzma
```

you might be able to do a 
```
try:
    lzma = compat.import_lzma()
except ImportError:
    lzma = None
```

then change the below to
```
getattr(lzma, 'LZMAFile', None)
```

any other ideas @WillAyd